src/Tribe/Views/V2/Template/Title.php

```diff
@@ -366,7 +366,7 @@ protected function build_day_title( $event_date ) {
 	 * @return string The built category archive title.
 	 */
 	protected function build_category_title( $title, $cat ) {
-		$new_title = '<a href="' . esc_url( tribe_get_events_link() ) . '">' . $title . '</a>';
```

Comment:
I think the category title still needs the link, we might need to figure out how to exclude it in the specific places we need it to be removed.

Reply:
@bordoni I don't think there's a use case for having a linked category on the page title. I can't find anywhere else where this title is being used. 
![Screenshot 2022-01-04 at 22 14 49](https://user-images.githubusercontent.com/22029087/148111819-92f50796-4091-4c70-8ed3-60b23de51276.jpg)

Reply:
Ahh i see that is good to know. You are correct here, we need revise the depth problem that stephen brought up.